v1/website/static/css/custom.css

```diff
@@ -63,8 +63,8 @@ table td:first-child > code {
   color: #fff;
   font-weight: bold;
   font-size: 24px;
-  margin-top: -40px;
```

Comment:
@endiliey I needed this to be there because the `mainContainer` has a padding of 40px. I was wondering why it appeared there even though I removed it before committing 😞 

Without this line, there's a white bar below the announcement.

<img width="1099" alt="Screen Shot 2019-03-20 at 8 26 32 PM" src="https://user-images.githubusercontent.com/1315101/54731968-83aa0200-4b4e-11e9-95cc-92fd8cdc2f0c.png">

Reply:
Seems that it displays differently on different browser/ resolution. Doesn’t appear in my browser 😂